lib/theme/widgets/hs_buttons.dart

```diff
@@ -296,19 +301,28 @@ class HsDropDownButton extends StatelessWidget {
   final VoidCallback onPressed;
   final String? icon;
   final TextStyle? labelStyle;
+  final String? placeholder;
+  final TextStyle? placeholderStyle;
+  final FocusNode? focusNode;
+  final ValueChanged? onFocusChange;
   const HsDropDownButton(
       {required this.onPressed,
       super.key,
       bool? isRequired,
       this.icon,
       this.value,
-      this.labelStyle});
+      this.labelStyle,
+      this.placeholder,
+      this.placeholderStyle,
+      this.focusNode,
+      this.onFocusChange});
   @override
   Widget build(BuildContext context) {
     return SecondaryButton(
-      // TODO: implement placeholder with enum of preriod
-      label: value ?? HatSpaceStrings.current.pleaseSelectValue,
-      labelStyle: labelStyle,
+      focusNode: focusNode,
+      onFocusChange: onFocusChange,
+      label: value ?? placeholder ?? HatSpaceStrings.current.pleaseSelectValue,
+      labelStyle: value == null ? placeholderStyle : labelStyle,
```

Comment:
The string to select value is specific to a single worker, it should not be used in a shared widget. Please change placeholder to be required